compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -583,6 +583,17 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         Ok(self.read_immediate(op)?.to_scalar())
     }
 
+    pub fn read_mem_place_meta(
```

Comment:
This doesn't seem worth it for a one-time operation. I think I'd prefer to see this inlined at the only call site.